### DISCUSSION TITLE
Pushing to registry using GITHUB_TOKEN fails one hour (Token seems expired)

### DISCUSSION THREAD

**Discussion by paresy** [community/community#25918] [Actions]
<p>As the title says everything was working fine, until i transferred the repo from my personal account to the organization account. Now the action workflow can’t push new images onto the registry. The login works but the action fails with: </p>
<p>Run docker login <a href="http://docker.pkg.github.com" rel="nofollow noopener">docker.pkg.github.com</a> -u xxxxx -p *** <br>
WARNING! Using --password via the CLI is insecure. Use --password-stdin.<br>
WARNING! Your password will be stored unencrypted in /home/runner/.docker/config.json.</p>
<p>Configure a credential helper to remove this warning. See<br>
<a href="https://docs.docker.com/engine/reference/commandline/login/#credentials-store" rel="nofollow noopener">https://docs.docker.com/engine/reference/commandline/login/#credentials-store</a><br>
Login Succeeded<br>
The push refers to repository [<a href="http://docker.pkg.github.com/xxxx/xxxx/build-xxxx" class="inline-onebox-loading" rel="nofollow noopener">docker.pkg.github.com/xxxx/xxxx/build-xxxx</a>]<br>
028bd0a4055f: Preparing<br>
a0482b2ad916: Preparing<br>
e55237605e99: Preparing<br>
c6638992f7d4: Preparing<br>
bc8b8b853a17: Preparing<br>
9762a95e7d15: Preparing<br>
562cf8447811: Preparing<br>
97041f29baff: Preparing<br>
9762a95e7d15: Waiting<br>
562cf8447811: Waiting<br>
97041f29baff: Waiting<br>
unauthorized: Your request could not be authenticated by the GitHub Packages service. Please ensure your access token is valid and has the appropriate scopes configured.<br>
##[error]Process completed with exit code 1.</p>
<p>When using a personal access token and using the Docker cli everything is fine. When an actions pushes an update to the same tag (not a new one) everything is fine aswell…</p>
<p>This is the workflow</p>
<pre><code>name: Build (xxxx1)

on: [deployment]

jobs:
  build-xxxx1:

    runs-on: ubuntu-latest

    if: github.event.deployment.environment == 'xxxx1'

    steps:
    - name: Checkout
      uses: actions/checkout@v1

    - name: Build
      run: |
        cd xxxx
        docker build --pull --build-arg xxxx=1 -t docker.pkg.github.com/xxxx/xxxx/build-xxxx1:latest -f Dockerfile .

    - name: Push
      run: |
        docker login docker.pkg.github.com -u ${{ github.actor }} -p ${{ secrets.GITHUB_TOKEN }}
        docker push docker.pkg.github.com/xxxx/xxxx/build-xxxx1:latest
</code></pre>
<p>This sounds like a permission bug.</p>
<p>Thanks for helping!</p>

**Comment by Yanjingzhu** [comment]
<p>Thank you for being here. I am afraid that I cannot reproduce your issue in my account.</p>
<p>Do you want to push to the current repo in organization? After transferring repo to organization, have you changed the owner name from your personal account to organization name when execute docker build and push command?   </p>
<pre><code>docker push docker.pkg.github.com/xxxx/xxxx/build-xxxx1:latest
</code></pre>
<p>If you are using a public repo, it would be better if you share the repo link here.  </p>

**Comment by paresy** [comment]
<p>Thanks for you answer.</p>
<p>Yes, i updated the repo link (from my personal to the org name).</p>
<p>Unfortunately this is a private repository which i cannot share public at the moment. Is there any place where i can submit this issue with details to my private repo?</p>
<p>Regards,</p>
<p>Michael</p>

**Comment by ethomson** [comment, marked as answer]
<p>Hi paresy, you’d need to contact support at <a href="https://support.github.com/" rel="nofollow noopener">https://support.github.com/</a></p>

**Comment by paresy** [comment]
<p>Done. Thanks!</p>

**Comment by eric-silverman** [comment]
<p>I’m hitting this as well, what was the resolution here?</p>

**Comment by paresy** [comment]
<p>Unfortunately nothing comfortable/feasible. The proposed workaround was to dump the image from docker, save it as an artifact, proceed to the next job (which will refresh the token), restore the image from artifacts, load it into docker and then push it to the registry.</p>
